City-of-Pori/pori-kada
# PORI-334 Added header and footer to error pages

To test:

1. ./build.sh update
2. drush updb -y && drush fra -y && drush cc all
3. Go to local.pori.fi/dsafghjkl and local.visitpori.fi/adsfghj and make sure the site error pages include top menu, main menu, language switcher, search and footer menu.
